plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -734,3 +734,9 @@ class Yarn2(
         val author: Set<String> = emptySet()
     )
 }
+
+private fun PackageJson.getScopeDependencies(type: YarnDependencyType) =
+    when {
+        type == YarnDependencyType.DEPENDENCIES -> dependencies
+        else -> devDependencies
+    }
```

Comment:
Better use

    when (type) {
        YarnDependencyType.DEPENDENCIES -> dependencies
        YarnDependencyType.DEV_DEPENDENCIES -> devDependencies
    }

Kotlin will recognize that the `when` is exhaustive, so no `else` is needed.